userManual/README.md

```diff
@@ -16,54 +16,115 @@ Explains:
+Just one place you can go to create a new slide to be saved into your Slides list! Where each slide is it's own event.
 
-####*You Mean There Are Other Places A New Slide Can Be Added?*
+#### *You Mean There Are Other Places A New Slide Can Be Added?*
 
 Yes! Another location you can go to create a new Slide is the Slides Page. Look at the bottom right corner, you should see a plus sign.
 Clicking this button takes you right back here, the Designer Page!
 
-![](slidesSnapshots/addSlide.PNG)
+![Add Slide Button](slidesSnapshots/addSlide.PNG)
 
-####*What does the content of the page look like?*
-![](designerSnapshots/page.PNG)
+#### *What does the content of the page look like?*
+![Designer Page](designerSnapshots/page.PNG)
 
 
 #### Adding a New Slide
 To add a new slide, as you may have noticed there are three required fields that must be filled in order to make a Slide.
-* Title of Event ![Title of Event](designerSnapshots/title.PNG)
-* Description of Event ![Description of Event](designerSnapshots/desc.PNG)
-* Photos ![Photos](designerSnapshots/photos.PNG)
+* Title of Event
 
-The date of the event is defaulted to the current day and the time is defaulted to the time you opened the page. 
+![Title of Event](designerSnapshots/title.PNG)
+* Description of Event
+
+![Description of Event](designerSnapshots/desc.PNG)
+* Photos (although this field does not have an asterisk it is a required field)
+
+![Description of Event](designerSnapshots/photos.PNG)
+
+##### Setting a Time and Date
+There is no default date, and if no date is selected then it is assumed that the Slide is a general announcement for the current day and the time is defaulted to the time you opened the page.
 
 ![](designerSnapshots/dateAndTime.PNG)
 
-To change the default date, select a different day on the calendar.
+To set a date date, select a day on the calendar.
 And to change the default time, click on the time and a clock will pop up:
 
 ![](designerSnapshots/changeTime.PNG)
 
-Select your new time and voila! To differentiate between AM and PM, click which option you want. 
+Select your new time and voila! To differentiate between AM and PM, click which option you want.
+
+##### Adding/Removing Photos on Slides
+Adding a photo to this application is much like any other application. 
+Click the plus button and find the photo you wish to show!
+Removing them is just as easy! Click the red 'X'.
+
+![Remove Photo](designerSnapshots/removePhoto.PNG)
+
+##### *I'm Finished Editing a Slide, Now What?*
+From this point on you can do a couple of things, click:
+* Clear, this option will clear your entire page
+
+![Clear](designerSnapshots/clear.PNG)
+* Preview, see what this does below
+
+![Preview](designerSnapshots/previow.PNG)
+* Submit, this option will bring you to the Preview page so you can see your slide. From here you will need to click submit again to make your Slide to be added to the Show View Page.
```

Comment:
__*Fake news!*__

Reply:
@KingMackle Right? 
Should this be considered a bug? Shouldn't adding a new slide either 
* Refresh the screen so all of the fields are empty
* Take you to the slides page to see your newly added slide
?